README.md

```diff
@@ -14,4 +14,5 @@ more than powerful analytics.
 1. Install the plugin with `ddev get valthebald/ddev-matomo`
 2. Review your database settings in .ddev/docker-compose.matomo.yaml
 3. Create matomo database if needed with `echo "SHOW TABLES;" | ddev import-db --database=${MATOMO_DATABASE_DBNAME}`'
-4. Access https://matomo.\<projectname\>.ddev.site in your browser and follow the installation steps.
+4. Restart DDEV to install DDEV Matomo with `ddev restart`
```

Comment:
I'm afraid I don't like this here, don't think it adds value.

Reply:
But if you proceed to the next step, will Matomo be ready without a `ddev restart`?

Reply:
@gitressa Indeed, ddev restart is required!